### PR TITLE
Fix logout

### PR DESCRIPTION
**Pull request by zachmargolis**
Builds on #8 

(just need to review the last commit only)

### CODE REVIEW

File: views/user_info.erb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <div class="p2 bg-green white clearfix">
   <% if userinfo %>
-    <form action="/auth/saml/logout" method="POST" class="right sm-show">
-      <button type="submit" class="btn p0 h6 white">Log out</button>
-    </form>
+    <div class="right sm-show">
+      <a href="/" class="btn p0 h6 white">Log out</button>
```
Comment (jessieay) [inline views/user_info.erb]: I am not clear on why going to the root will log someone out? Or is this just to make the link work but not actually function to do the logout stuff?

Comment (zachmargolis) [inline views/user_info.erb]: This is how it works. 😬  The app currently doesn't maintain any sort of session or cookie or anything. Being "logged in" is having an authorization code in the URL, so linking back to the home page without a param shows a logged out page #hacks

File: spec/app_spec.rb [hunk 2]
```diff
@@ -62,5 +62,14 @@
...
       expect(last_response.body).to include(email)
     end
+
+    it 'has a logout link back to root' do
+      get '/auth/result', code: code
+
+      doc = Nokogiri::HTML(last_response.body)
+
+      logout_link = doc.at('a[href="/"]')
+      expect(logout_link.text.strip).to eq('Log out')
```
Comment (jessieay) [inline spec/app_spec.rb]: is there anything else we can look for to ensure that visiting this url makes the user be logged out?

Comment (zachmargolis) [inline spec/app_spec.rb]: So this is just checking the rendered page, not following it and visiting the page...I guess I could do a `get` to that location and then make sure the email isn't there anymore....will update momentarily

Comment (zachmargolis) [inline spec/app_spec.rb]: See e388a3d9d868659e3669e85ec4bca351c6b031af

**Pull request merged**